internal/outputs/documentation.go

```diff
@@ -36,13 +38,16 @@ func NewMarkdownPolicyDocumentation(policies []*policy.Policy) PolicyDocumentati
 }
 
 func (m *MarkdownPolicyDocumentation) GenerateDocumentation() string {
-
+	sort.SliceStable(m.policies, func(i, j int) bool {
+		return m.policies[i].Group < m.policies[j].Group
+	})
 	var sb strings.Builder
 
-	sb.WriteString("# Available Policies\n\n|Title|Description|Group|File|\n|-|-|-|-|")
+	sb.WriteString("# Available Policies\n\n|Group|`Title|Description|File|\n|-|-|-|-|")
 
 	for _, p := range m.policies {
-		sb.WriteString(fmt.Sprintf("\n |%s|%s|%s|%s|", p.Title, p.Description, p.Group, p.File))
+		policyFileURL := fmt.Sprintf("%s/blob/%s/%s", config.DefaultGitRepository, config.DefaultGitBranch, p.File)
```

Comment:
maybe it is better idea to base on local directory - to be able to regenerate documentation while adding new policy?

Reply:
Good idea, i've added possibility to pass `--local-policy-dir` to generate command and added script + workflow to check.
Anyway the link here I left, as the main use case is to generate documentation for our Github repo.